wts/blueprints/aggregate.py

```diff
@@ -0,0 +1,72 @@
+import asyncio
+import flask
+import httpx
+import requests
+import time
+
+from urllib.parse import urljoin, urlparse
+
+from ..auth import async_login_required
+from ..models import db, RefreshToken
+from ..tokens import async_get_access_token
+
+
+blueprint = flask.Blueprint("aggregate", __name__)
+
+
+#  TODO add swagger doc for this endpoint
+#  TODO add tests
+@blueprint.route("/authz", methods=["GET"])
+@async_login_required
+async def get_aggregate_authz():
```

Comment:
could we design the `/aggregate` endpoint in a generic way?
`/aggregate/user/user` would hit all the the `/user/user` endpoints and aggregate all the results.
and you could use `/aggregate/<some other endpoint>` without having to write new code.
if we only want the `authz` field of the userinfo response, that could be configuration or a query parameter (or hardcoded, i guess)